### ISSUE TITLE
Domain Info and domain on Django admin pull portfolio info when applicable

### ISSUE THREAD

**Issue by abroddrick** [cisagov/manage.get.gov#2975]
### Issue description

For org model Domains/ Domain info, for all portfolio specific fields we should be pulling from the portfolio instead of the domain info object. For example, organization name should come from the portfolio, as changes should be done on the portfolio not on domain info. However, we only want to perform these changes on the django admin tables, all exports (domain exports, analytics pages exports), and any analytics graphs where this applicable. 

### Acceptance criteria

- [ ] Follow what was done in #2852 
- [ ] Domain Info/Domain fields on django admin should be have a corresponding function that is returning either the stored domain info value or the corresponding portfolio value where appropriate. 
- [ ] Filters, sorts and table views on domain info &domain on Django admin are all updated to now pull from the portfolio when needed
- [ ] All reports are updated to use the appropriate fields (this includes those on the analytics page and on the tables) 
- [ ] Double check all domain request reports are also using similar logic and update accordingly
- [ ] Do not change the change form (domain/domain info record) view

### Additional context
implementation suggestion note: we should be have a corresponding @property function  for each domain info field that is used for reports/analytics/ django admin table/django admin filter. Below are  Domain Info/Domain fields but you will not need to handle each one, only create @property fields **IF** we are using this value for a report/ analytics board or to display on a table:
  - [ ]  organization_name
  - [ ]  generic_org_type  (called "organization_type" in Portfolio)
  - [ ]  federal_agency - should come from federal agency from federal agency entry associated with the portfolio
  - [ ]  federal_type - should come from federal agency table entry associated with the portfolio
  - [ ]  senior_official -should come from portfolio
  - [ ]  address_line1 - should be from portfolio organization address
  - [ ]  address_line2 - should be from portfolio organization address
  - [ ]  state_territory - should be from portfolio organization address
  - [ ]  zipcode - should be from portfolio organization address
  - [ ]  urbanization - should be from portfolio organization address

### Links to other issues

- blocked by #2852 
- blocks #2867

**Comment by abroddrick** [comment]
Do to dev huddles today this one is getting closed as a won't do based on finding from the pre-requesite tickets. Aspects of this ticket's AC (that are still applicable) will go into #2867 .

**Comment by abroddrick** [comment]
After discussing with dev, this ticket is being re-opened but edited to now reflect the ACs that were already completed. #2867 will still contain some of the original ACs that were part of this ticket
